unitpackage/collection.py

```diff
@@ -251,3 +251,20 @@ def __getitem__(self, identifier):
                 f"The collection has more than one entry with identifier '{identifier}'."
             )
         return entries[0]
+
+    def save_entries(self, outdir=None):
```

Comment:
We might want to add possibilities to save selected entries in specific subfolders.

We could define an `outdir_pattern` which creates an outdir from descriptors within an entry.

```python
def pattern(outdir, entry):
    return os.path.join(outdir, entry.user) # stores entries based on the user who created the package

def save_entries(self, outdir=None, outdir_pattern=pattern):
    for entry in self:
        if outdir_pattern:
            outdir = outdir_pattern(outdir, entry)
            entry.save(outdir=outdir)
``` 

We could implement a similar approach for the filename.